docs/build/tools/netrunner.md

```diff
@@ -0,0 +1,189 @@
+# Network Runner
+
+The Avalanche Network Runner tool allows a user to define, create and interact with a network of Avalanche nodes. It can be used for development and testing.
```

Comment:
```suggestion
The Avalanche Network Runner allows a user to define, create and interact with a network of Avalanche nodes. It can be used for development and testing.
```